packages/lesswrong/server/migrations/index.ts

```diff
@@ -87,6 +87,7 @@ import './2021-12-02-fillHideFromAuthorPage'
 import './2021-12-02-fillReviewCountField'
 import './2021-12-13-updateQuadraticVotes'
 import './2022-01-30-updateFinal2020ReviewVotes'
+import './2022-02-21-populateLinkSharingKeys'
```

Comment:
May need to run this migration?  Not clear - seems like it's backfilling keys for "share with" links for draft posts(?), which may not exist absent the collaborative editor mode

Reply:
Does not need to be run for EA forum (running it should be harmless though). Only necessary for posts that were link-shared before link-sharing keys were introduced into the codebase.